x-pack/platform/plugins/shared/onechat/server/services/agents/modes/chat/run_chat_agent.ts

```diff
@@ -27,13 +32,26 @@ export type RunChatAgentFn = (
  * Create the handler function for the default onechat agent.
  */
 export const runChatAgent: RunChatAgentFn = async (
-  { nextInput, conversation = [], tools = [], runId = uuidv4(), systemPrompt },
-  { logger, request, modelProvider, events }
+  {
+    nextInput,
+    conversation = [],
+    toolSelection = [{ toolIds: ['*'] }],
```

Comment:
nit:  we are using `[{ toolIds: ['*'] }]`  4 times as default args, maybe good time to have
`const ALL_TOOLS = [{ toolIds: ['*'] }]` or at least `allToolsSelectionWildcard` from onechat-common